docs/connecting-data/connecting-to-apis-and-external-data/jira.md

```diff
@@ -1,36 +1,161 @@
 
 - A unique name for your data source connection to be used in queries
 - The domain of your Atlassian account
 - The email associated with your Atlassian account
 - The API token generated through your Atlassian account
 
-### Form
+### Data Source Form
+To locate the JIRA form, follow the steps in [Connecting Your Data to DataDistillr](../../). When you get to the window to choose the data source type, select API as shown below.
+
+<figure markdown>
+  ![Data Source Wizard][image-7]{ width="100%" }
+  <figcaption>Data Source Wizard</figcaption>
+</figure>
+
+
+
+On the API screen, select JIRA from the list of API forms.
+
+<figure markdown>
+  ![List of APIs][image-8]{ width="100%" }
+  <figcaption>List of APIs</figcaption>
+</figure>
+
+
 
-![JIRA Form][image-1]
+The following form will appear. Instructions can be found below on how to find the information requried to fill each field on the JIRA API form.
 
+Once you have filled out all the fields, press the green 'Save' button, and your API will be connected!
+
+<figure markdown>
+  ![JIRA Form][image-1]{ width="100%" }
+  <figcaption>JIRA Form</figcaption>
+</figure>
+
+
+### Name
+Enter any name that will help you recognize this data source from within your query window.
+
+Acceptable characters include:
+
+- lowercase alphanumeric characters
+- underscores
 
 ### Domain
-You can access your Organization at [https://admin.atlassian.com/](https://admin.atlassian.com/) to find your domain.
+An Organization is a management layer that gives admins the ability to view and apply controls to all Atlassian accounts using an email address belonging to their company.
+
+!!! example "Steps for getting your domain"
+
+    === "1. Home page"
+
+        In the JIRA app, click your profile icon in the top right corner of the page.
 
-![Finding your domain][image-3]
+        ![JIRA Home Page][image-10]
 
+    === "2. Accout settings"
+
+        In the section with your name, click the **Account settings** link, this will take you to another page that contains information about your Atlassian account.
+
+        ![Account Pop Up][image-11]
+
+    === "3. Find Domain"
+
+        In the left-hand sidebar, click **Products** and there will be your domain under the "Jira settings" section.
+
+        ![Find Domain][image-12]
 
 ### User
+This is the email that is tied to your Atlassian account.
+
+!!! example "Steps for getting your email"
+
+    === "1. Home page"
+
+        In the JIRA app, click your profile icon in the top right corner of the page.
+
+        ![JIRA Home Page][image-10]
+
+    === "2. Accout settings"
+
+        In the section with your name, click the **Account settings** link, this will take you to another page that contains information about your Atlassian account.
+
+        ![Account Pop Up][image-11]
+
+    === "3. Find email"
+
+        In the left-hand sidebar, click **Products** and there will be your domain under the "Jira settings" section.
```

Comment:
click **Products** and your domain will be found - wording feels a little awkward

Reply:
Also, replace domain with email and "Jira settings" with "Current email"